pom.xml

```diff
@@ -388,7 +388,13 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>structs</artifactId>
-      <version>1.4</version>
+      <version>1.7</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>credentials</artifactId>
```

Comment:
why do we need this now?

Reply:
it is a dependency to force because of upgrade.
Please note the test scope.

Reply:
👍